ui/about.ui

```diff
@@ -2,7 +2,8 @@
   <template class="MidoriAbout" parent="GtkAboutDialog">
     <property name="modal">yes</property>
     <property name="logo-icon-name">midori</property>
-    <property name="copyright">2007-2018 Christian Dywan</property>
+    <property name="copyright">2019 Fundación Astian</property>
+    <property name="copyright">Creator and Founder Christian Dywan</property>
```

Comment:
<property name="copyright">Copyright 2007-2019 Astian Foundation</property>

- We can't define the same property twice.
- It's important to **keep the timeline** here so it doesn't look as if the project was started this year.
- Render the name of the foundation **in English** to make it more inclusive - we could use a translatable string as well, though (`translatable="yes"`).
- Consider `"authors"` to list proper names of the most notable contributors.